### PR TITLE
NRPT-10: Cleanup repo

### PR DESCRIPTION
**Pull request by dhlevi**
Moved static content into static folders, organized pipes, cleanup of some unused files (note, openshift template cleanup in a different ticket). Updates for some module paths, removed doc search modules.

See ticket [NRPT-10](https://bcmines.atlassian.net/browse/NRPT-10)

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
+import { LifecycleComponent } from 'app/static-pages/lifecycle/lifecycle.component';
+import { ReclamationComponent } from 'app/static-pages/reclamation/reclamation.component';
+import { TailingsManagementComponent } from 'app/static-pages/tailings-management/tailings-management.component';
+import { TopicsOfInterestComponent } from 'app/static-pages/topics-of-interest/topics-of-interest.component';
+import { WaterQualityComponent } from 'app/static-pages/water-quality/water-quality.component';
 import { MainMapComponent } from 'app/map/main-map/main-map.component';
-import { SearchComponent } from 'app/search/search.component';
 import { NotFoundComponent } from './not-found/not-found.component';
-import { EnforcementActionsComponent } from './enforcement-actions/enforcement-actions.component';
+import { EnforcementActionsComponent } from './static-pages/enforcement-actions/enforcement-actions.component';
```
Comment (KitArmstrong) [inline src/app/app-routing.module.ts:15]: This line is using a relative path for importing where the ones above are using an absolute path. Wonder if we should select one way and update all files to follow that approach? Not sure if that is too much work for this ticket though.

File: src/app/projects/project-detail/authorizations/authorizations-tab-content.component.spec.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { AuthorizationsTabContentComponent } from './authorizations-tab-content.
 import { ActivatedRoute } from '@angular/router';
 import { Project } from '../../../models/project';
 
-import { OrderByPipe } from '../../../filters/order-by.pipe';
+import { OrderByPipe } from '../../../pipes/filters/order-by.pipe';
```
Comment (KitArmstrong) [inline src/app/projects/project-detail/authorizations/authorizations-tab-content.component.spec.ts:10]: Absolute paths get rid of these sorts of things.

**Comment by KitArmstrong** [comment]
Nice work. That cleaned it up nicely and will be a much better starting place. Just the comment about absolute vs relative paths.

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (dhlevi) [reply src/app/app-routing.module.ts:15]: Yeah, it's kind of a mish-mash all over the place. There's a mix of absolute and relative throughout. I'm happy with going absolute paths over relative. I can certainly make that change now... there's a lot of them but it shouldn't take too long to update.

**Pull request merged**